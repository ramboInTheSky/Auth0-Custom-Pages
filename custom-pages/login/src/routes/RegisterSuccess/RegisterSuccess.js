import React from 'react'
import Alert from '@nice-digital/nds-alert'

// local imports
import { hideNav } from '../../util'

// style
import './RegisterSuccess.scss'

export const RegisterSuccess = () => {
  hideNav()
  return (
    <div>
      <h3>Thank you</h3>
      <h5>
        We've sent you an email with an activation link. To verify your details
        and start using your account, click on the link.
      </h5>
      <Alert type="caution">
        Note: check your spam folder if you do not receive the activation email.
      </Alert>
      <p>
        <a href="#">&larr; Take me back to Login</a>
      </p>
    </div>
  )
}

export default RegisterSuccess
