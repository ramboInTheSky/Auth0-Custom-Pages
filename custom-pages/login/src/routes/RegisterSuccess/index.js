import RegisterSuccess from './RegisterSuccess'

export default {
  path: 'regsuccess',
  component: RegisterSuccess
}
