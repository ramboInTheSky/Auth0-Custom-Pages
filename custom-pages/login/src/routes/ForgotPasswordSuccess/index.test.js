import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-14'
import RegisterSuccess from './'

Enzyme.configure({ adapter: new Adapter() })

describe('RegisterSuccess components', () => {
  it('should render <RegisterSuccess /> correctly', () => {
    expect(shallow(<RegisterSuccess.component />)).toMatchSnapshot()
  })
})
