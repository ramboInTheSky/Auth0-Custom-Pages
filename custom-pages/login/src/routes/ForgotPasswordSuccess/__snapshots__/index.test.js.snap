// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterSuccess components should render <RegisterSuccess /> correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ForgotPasswordSuccess />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Success
        </h3>,
        <h5
          data-qa-sel="forgotPasswordSuccess-message1"
        >
          If that email address is in our records, we will send an email to you containing instructions on how to reset your password. You may have to wait a few minutes for the email. If you can’t find the email, please check your spam and deleted items folders.
        </h5>,
        <h5
          data-qa-sel="forgotPasswordSuccess-message2"
        >
          Please act on the email straight away.
        </h5>,
        <Link
          data-qa-sel="forgotPasswordSuccess-link-to-login"
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/"
        >
           Return to Sign in 
        </Link>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Success",
        },
        "ref": null,
        "rendered": "Success",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "If that email address is in our records, we will send an email to you containing instructions on how to reset your password. You may have to wait a few minutes for the email. If you can’t find the email, please check your spam and deleted items folders.",
          "data-qa-sel": "forgotPasswordSuccess-message1",
        },
        "ref": null,
        "rendered": "If that email address is in our records, we will send an email to you containing instructions on how to reset your password. You may have to wait a few minutes for the email. If you can’t find the email, please check your spam and deleted items folders.",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Please act on the email straight away.",
          "data-qa-sel": "forgotPasswordSuccess-message2",
        },
        "ref": null,
        "rendered": "Please act on the email straight away.",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": " Return to Sign in ",
          "data-qa-sel": "forgotPasswordSuccess-link-to-login",
          "onlyActiveOnIndex": false,
          "style": Object {},
          "to": "/",
        },
        "ref": null,
        "rendered": " Return to Sign in ",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Success
          </h3>,
          <h5
            data-qa-sel="forgotPasswordSuccess-message1"
          >
            If that email address is in our records, we will send an email to you containing instructions on how to reset your password. You may have to wait a few minutes for the email. If you can’t find the email, please check your spam and deleted items folders.
          </h5>,
          <h5
            data-qa-sel="forgotPasswordSuccess-message2"
          >
            Please act on the email straight away.
          </h5>,
          <Link
            data-qa-sel="forgotPasswordSuccess-link-to-login"
            onlyActiveOnIndex={false}
            style={Object {}}
            to="/"
          >
             Return to Sign in 
          </Link>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Success",
          },
          "ref": null,
          "rendered": "Success",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "If that email address is in our records, we will send an email to you containing instructions on how to reset your password. You may have to wait a few minutes for the email. If you can’t find the email, please check your spam and deleted items folders.",
            "data-qa-sel": "forgotPasswordSuccess-message1",
          },
          "ref": null,
          "rendered": "If that email address is in our records, we will send an email to you containing instructions on how to reset your password. You may have to wait a few minutes for the email. If you can’t find the email, please check your spam and deleted items folders.",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Please act on the email straight away.",
            "data-qa-sel": "forgotPasswordSuccess-message2",
          },
          "ref": null,
          "rendered": "Please act on the email straight away.",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": " Return to Sign in ",
            "data-qa-sel": "forgotPasswordSuccess-link-to-login",
            "onlyActiveOnIndex": false,
            "style": Object {},
            "to": "/",
          },
          "ref": null,
          "rendered": " Return to Sign in ",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFourteenAdapter {
      "options": Object {
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "prevContext": true,
          },
          "getChildContext": Object {
            "calledByRenderer": true,
          },
        },
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
